init/gstd.in

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides: GstD
+# Short-Description: start and stop GstD
+# Description: Gstreamer Daemon is a framework for
+#              controlling audio and video streaming
+### END INIT INFO
+
+manage_dir()
+{
+    dir="$1";
+    mode="$2";
+    user="$3";
+
+    #Create directory if not created yet
+    mkdir -m "$mode" -p "$dir";
+    if [ $? -ne 0 ]; then
+        echo "GstD init: Error, cannot create directory $dir" 1>&2;
+        exit 1
+    fi
+
+    #Set the owner
+    chown "$user" "$dir";
```

Comment:
I am wondering if we need to make this recursive. What if the dir already existed and had files in it?

Reply:
done